docs/src/main/asciidoc/writing-native-applications-tips.adoc

```diff
@@ -139,38 +219,115 @@ method com.amazonaws.services.s3.model.CryptoConfiguration.<init>(CryptoMode)
 Call path from entry point to com.amazonaws.services.s3.model.CryptoConfiguration.<init>(CryptoMode):
 ----
 
-If one were to execute the native executable generation process manually, the `--delay-class-initialization-to-runtime=com.amazonaws.services.s3.model.CryptoConfiguration` flag would need to be passed to `native-image` to configure runtime initialization for `CryptoConfiguration`.
+If you need to delay the initialization of a class, you can use the `--initialize-at-run-time=<package or class>` configuration knob.
 
-Quarkus simplifies things by allowing extensions authors to simply register a `RuntimeInitializedClassBuildItem`. A simple example of doing so could be:
+It should be added to the `native-image` configuration using an `<additionalBuildArg>` as for the examples above.
```

Comment:
`as for the examples above` -> `as shown in the examples above`?

Reply:
Yes, better, applied.